packages/connectivity/connectivity_platform_interface/lib/method_channel_connectivity.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+
+import 'connectivity_platform_interface.dart';
+
+@visibleForTesting
+const MethodChannel method_channel = MethodChannel('plugins.flutter.io/connectivity');
```

Comment:
nit: Technically this also needs documentation. It would be fine to just have the documentation say this is test only and not to use it.

Reply:
This will actually be used once I make the change in `connectivity.dart`.  Added a comment anyways :)